onnxruntime/hosting/server/http_session.h

```diff
@@ -154,6 +153,7 @@ class http_session : public std::enable_shared_from_this<http_session> {
   }
 };
 
-} // namespace onnxruntime
+}  // namespace hosting
+}  // namespace onnxruntime
 
 #endif  //BEAST_SERVER_HTTP_SESSION_H
```

Comment:
This macro seems not match the file path and name.

Reply:
Because of the underscore?

Reply:
Oh because it has BEAST in it. I'll need to change that for all of them.

Reply:
Here is the [coding convention](https://google.github.io/styleguide/cppguide.html#The__define_Guard):

> To guarantee uniqueness, they should be based on the full path in a project's source tree. 

We may not need to have full path, but "BEAST" is not in the path. So we should not use `BEAST` in the name.

Reply:
Fixed